.github/workflows/ci.yml

```diff
@@ -133,7 +139,17 @@ jobs:
     - name: Build
       shell: bash
       run: cmake --build build --config ${{ matrix.type.name == 'Debug' && 'Debug' || 'Release' }} --target install
-      
+
+    - name: Build Android example
+      if: matrix.platform.name == 'Android'
+      shell: bash
+      run: |
+        cd examples/android
+        ARCH_ABI=$(echo ${{ matrix.config.flags }} | sed 's/.*-DCMAKE_ANDROID_ARCH_ABI=\([[:graph:]]*\).*/\1/')
+        sed "s/abiFilters.add(\"[[:alnum:]_-]*\")/abiFilters.add(\"${ARCH_ABI}\")/g" -i app/build.gradle.kts
+        sed "s/ndkVersion = \"[[:digit:].]*\"/ndkVersion = \"25.2.9519653\"/g" -i app/build.gradle.kts
```

Comment:
Why are all these find/replace `sed` calls necessary?

Reply:
The settings in build.gradle.kts need to be changed. The architecture to use (e.g. x86) is hardcoded in the example, but the CI needs to test different architectures. I'm open for alternatives on this. I do realize that the last two sed commands are harder than they need to be, I'll change them into something slightly simpler.

The ndkVersion technically doesn't need to be changed. It is already the same on the CI and in the example, so the last sed command can be removed. The question is: will you guarantee that the example remains in sync with the NDK version used on the CI, or will you allow them to be independent from each other?

Reply:
> but the CI needs to test different architectures

Is it worth testing on all architectures versus just 1? Is it possible that one of these Gradle jobs fails but the others don't? If they either all pass or all fail then we can maybe get away with a single one of them. I'd rather not fight with Gradle.

Reply:
> Is it worth testing on all architectures versus just 1?

Probably not. If we only test one, and I can't change the example with sed commands, then the example will just have to build with the exact same ABI and NDK as is tested by the CI. That's not really an issue, it's just something to keep in mind if the CI or example are ever changed later.

Reply:
> the example will just have to build with the exact same ABI and NDK as is tested by the CI

I think I'm cool with that. I'd rather deal with that then have to patch the build scripts to make CI work.